helm/blueapi/values.yaml

```
@@ -117,6 +117,11 @@ worker:
     repositories: []
     # - name: "dodal"
     #   remote_url: https://github.com/DiamondLightSource/dodal.git
+  logging:
+      graylog_enabled: False
+      graylog_host: "graylog-log-target.diamond.ac.uk"
+      graylog_port: 12232
```

Nit: May as well group rather than prefix

```suggestion
      graylog:
        enabled: False
        host: "graylog-log-target.diamond.ac.uk"
        port: 12232
```

---

Implemented